metadata-ingestion-modules/airflow-plugin/tests/conftest.py

```diff
@@ -1,15 +1,14 @@
 import pathlib
 import site
 
+from datahub.testing.pytest_hooks import (  # noqa: F401,E402
```

Comment:
is ignoring `E402` required?
https://docs.astral.sh/ruff/rules/module-import-not-at-top-of-file/

here specifically and everywhere this code snippet is being used..

Reply:
yup - will fix in a follow up PR